administrator/language/de-DE/plg_multifactorauth_email.ini

```
@@ -0,0 +1,28 @@
+; Note : All ini files need to be saved as UTF-8
+
+PLG_MULTIFACTORAUTH_EMAIL="Multi-Faktor-Authentifizierung - Authentifizierungscode per E-Mail"
+PLG_MULTIFACTORAUTH_EMAIL_CONFIG_FORCE_ENABLE_DESC="Soll der Authentifizierungscode per E-Mail automatisch als Option für <em>alle</em> Benutzer hinzugefügt werden? Es wäre praktisch den Benutzern, die keinen Zugang mehr zu ihrem Hauptauthentifikator haben und keine Kopie der Backup-Codes aufbewahrt haben, eine Alternative zu bieten, <strong>allerdings auf Kosten eines geringeren Kontroll- und Sicherheitsstandards</strong>."
+PLG_MULTIFACTORAUTH_EMAIL_CONFIG_FORCE_ENABLE_LABEL="Aktivieren erzwingen"
+PLG_MULTIFACTORAUTH_EMAIL_CONFIG_TIMESTEP_120="Zwei Minuten (empfohlen)"
+PLG_MULTIFACTORAUTH_EMAIL_CONFIG_TIMESTEP_180="Drei Minuten"
+PLG_MULTIFACTORAUTH_EMAIL_CONFIG_TIMESTEP_300="Fünf Minuten"
+PLG_MULTIFACTORAUTH_EMAIL_CONFIG_TIMESTEP_30="Eine halbe Minute"
+PLG_MULTIFACTORAUTH_EMAIL_CONFIG_TIMESTEP_60="Eine Minute"
+PLG_MULTIFACTORAUTH_EMAIL_CONFIG_TIMESTEP_DESC="Der Code wird im gewählten Zeitraum neu erstellt. Bitte beachten, dass ein generierter Code mindestens so lange und höchstens doppelt so lange gültig ist. Je länger diese Zeitspanne ist, desto wahrscheinlicher ist es, dass der Code manipuliert werden kann und desto unsicherer ist die Website. Ein Zeitraum von 2 Minuten ist ein guter Kompromiss zwischen Benutzerfreundlichkeit und Sicherheit."
+PLG_MULTIFACTORAUTH_EMAIL_CONFIG_TIMESTEP_LABEL="Zeitraum der Code-Erzeugung"
+PLG_MULTIFACTORAUTH_EMAIL_EMAIL_BODY="Multi-Faktor-Authentifizierung auf {SITENAME}. Der Authentifizierungcode lautet {CODE}."
+PLG_MULTIFACTORAUTH_EMAIL_EMAIL_SUBJECT="Der {SITENAME} - Authentifizierungscode lautet -{CODE}-"
+PLG_MULTIFACTORAUTH_EMAIL_ERR_INVALID_CODE="Ungültiger oder abgelaufener Code. Bitte die Seite aktualisieren, um einen neuen Code anzufordern. Sicherstellen, dass der Code innerhalb von zwei Minuten nach der Anforderung eingegeben wird."
+PLG_MULTIFACTORAUTH_EMAIL_LBL_DISPLAYEDAS="Authentifizierungscode per E-Mail"
+PLG_MULTIFACTORAUTH_EMAIL_LBL_LABEL="Authentifizierungscode"
+PLG_MULTIFACTORAUTH_EMAIL_LBL_PRE_MESSAGE="Ein sechsstelliger Code für die Multi-Faktor-Authentifizierung wurde per E-Mail gesendet. Bitte diesen unten eingeben."
+PLG_MULTIFACTORAUTH_EMAIL_LBL_SETUP_PLACEHOLDER="Sechsstelliger Authentifizierungscode"
+PLG_MULTIFACTORAUTH_EMAIL_LBL_SHORTINFO="Sechsstelligen Code per E-Mail erhalten"
+PLG_MULTIFACTORAUTH_EMAIL_MAIL_LBL="Joomla: Authentifizierungscode per E-Mail"
+PLG_MULTIFACTORAUTH_EMAIL_MAIL_MAIL_DESC="Wird bei Verwendung der Option „Authentifizierungscode per E-Mail“ von der Seite der Multi-Faktor-Authentifizierung an die Benutzer verschickt."
+PLG_MULTIFACTORAUTH_EMAIL_MAIL_MAIL_TITLE="Authentifizierungscode per E-Mail gesendet"
+PLG_MULTIFACTORAUTH_EMAIL_XML_DESCRIPTION="Verwendung eines zeitlich begrenzten, sechsstelligen Sicherheitscodes, der per E-Mail verschickt wird"
```

```suggestion
PLG_MULTIFACTORAUTH_EMAIL_XML_DESCRIPTION="Verwendung eines zeitlich begrenzten, sechsstelligen Sicherheitscodes, der per E-Mail verschickt wird."
```

+ verschicken vs. versenden